src/pages/Profile/Profile.ts

```diff
@@ -4,28 +4,112 @@ import Text from "../../components/Text/Text";
+import router from "../../index";
+import Store from "../../services/Store/Store";
+import IUser from "../../services/Store/store-interfaces/IUser";
+import UserRequest from "../../utils/api-interfaces/users/change-profile";
+import UserAPIController from "../../utils/controllers/user-api-controller";
+import PasswordRequest from "../../utils/api-interfaces/users/change-profile-password";
+import Button from "../../components/Button/Button";
 
 export default class Profile extends Component {
 
 	constructor() {
 
+		const user = Store.getState().user as IUser;
+
 		const goBack = new GoBack({});
-		const profileTitle = new Text({ text: "Иван" });
+		const profileTitle = new Text({text: user.first_name});
+
+		const email = new ProfileDataEntry({title: "Почта", type: "email", placeholder: user.email});
+		const login = new ProfileDataEntry({title: "Логин", type: "login", placeholder: user.login});
+		const firstName = new ProfileDataEntry({title: "Имя", type: "name", placeholder: user.first_name});
+		const lastName = new ProfileDataEntry({title: "Фамилия", type: "name", placeholder: user.second_name});
+		const userName = new ProfileDataEntry({title: "Имя в чате", type: "text", placeholder: user.display_name});
+		const phoneNumber = new ProfileDataEntry({title: "Телефон", type: "tel", placeholder: user.phone});
+		const oldPassword = new ProfileDataEntry({title: "Старый пароль", type: "password", placeholder: "Скрыто"});
+		const newPassword = new ProfileDataEntry({title: "Новый пароль", type: "password", placeholder: "Пусто"});
+
+		const updateUserDataAction = new ProfileActionsEntry({
+			text: "Изменить данные",
+			action: () => {
+
+				const inputs = this._element.querySelectorAll('input');
+
+				let userData: UserRequest = {
+					email: inputs[0].value || inputs[0].placeholder,
+					login: inputs[1].value || inputs[1].placeholder,
+					first_name: inputs[2].value || inputs[2].placeholder,
+					second_name: inputs[3].value || inputs[3].placeholder,
+					display_name: inputs[4].value || inputs[4].placeholder,
+					phone: inputs[5].value || inputs[5].placeholder,
+				};
 
-		const email = new ProfileDataEntry({ title: "Почта", type: "email", placeholder: "pochta@yandex.ru" });
-		const login = new ProfileDataEntry({ title: "Логин", type: "login", placeholder: "ivanivanov" });
-		const firstName = new ProfileDataEntry({ title: "Имя", type: "name", placeholder: "Иван" });
-		const lastName = new ProfileDataEntry({ title: "Фамилия", type: "name", placeholder: "Иванов" });
-		const userName = new ProfileDataEntry({ title: "Имя в чате", type: "text", placeholder: "Иван" });
-		const phoneNumber = new ProfileDataEntry({ title: "Телефон", type: "tel", placeholder: "+7 (909) 967 30 30" });
+				new UserAPIController().changeProfile(userData)
+					.then(() => Store.set('user', userData))
+					.then(() => {
+						inputs.forEach(
+							input => {
+								input.value = "";
+							}
+						)
```

Comment:
Этот код можно сделать более читаемым вот так
```suggestion
   const clear = (input: HTMLInputELement) => {
     input.value = ""
   }
   inputs.forEach(clear)
```